changelogs/fragments/redhat_subscribtion.yml

```diff
@@ -2,4 +2,6 @@ bugfixes:
   - redhat_subscription - made code compatible with Python3 (https://github.com/ansible/ansible/pull/58665)
 minor_changes:
   - redhat_subscription - do not call ``subscribtion-manager`` command, when it is not necessary (https://github.com/ansible/ansible/pull/58665)
-  - redhat_subscription - made code more testable (https://github.com/ansible/ansible/pull/58665)
\ No newline at end of file
+  - redhat_subscription - made code more testable (https://github.com/ansible/ansible/pull/58665)
+major_changes:
```

Comment:
Done

Reply:
Please rename this file rather than modifying the existing fragment file from a previous PR. We need separate fragment files _per PR_ to ensure we have all the changes needed to generate an accurate changelog.

Reply:
The names of the file do not matter and can be anything so long as they are unique.

Reply:
Done